modules/setting/repository.go

```diff
@@ -325,4 +329,8 @@ func newRepository() {
 	if !filepath.IsAbs(Repository.Upload.TempPath) {
 		Repository.Upload.TempPath = path.Join(AppWorkPath, Repository.Upload.TempPath)
 	}
+
+	repoAvatarSec := Cfg.Section("repo-archive")
+	storageType := repoAvatarSec.Key("STORAGE_TYPE").MustString("")
+	RepoArchive.Storage = getStorage("repo-archive", storageType, repoAvatarSec)
```

Comment:
```suggestion
	repoArchiveSec := Cfg.Section("repo-archive")
	storageType := repoAvatarSec.Key("STORAGE_TYPE").MustString("")
	RepoArchive.Storage = getStorage("repo-archive", storageType, repoArchiveSec)
```

Reply:
we may should mention it in the docs, at least a bit in `Configuration Cheat Sheet`

Reply:
Docs added.